plugins/inx/server_utxo.go

```diff
@@ -333,8 +333,21 @@ func (s *INXServer) ListenToLedgerUpdates(req *inx.MilestoneRangeRequest, srv in
 	}
 
 	sendFunc := func(task *workerpool.Task, index iotago.MilestoneIndex) error {
-		newOutputs := task.Param(1).(utxo.Outputs)
-		newSpents := task.Param(2).(utxo.Spents)
+		newOutputs, ok := task.Param(1).(utxo.Outputs)
```

Comment:
better to use const/enums instead of magic numbers for the Index of `task.Param`